dspace-api/src/main/java/org/dspace/authorize/AuthorizeServiceImpl.java

```diff
@@ -942,4 +945,40 @@ private String formatCustomQuery(String query) {
             return query + " AND ";
         }
     }
+
+    public DiscoverResult retrieveObjectWithAdmin(Context context, DiscoverQuery discoverQuery,
```

Comment:
If we are keeping this method, it needs an `@Override` annotation here (to link it to its JavaDocs).  We also would need to add automated tests for this method to `AuthorizeServiceTest`